### PR TITLE
style: change code style

### PR DESCRIPTION
**Pull request by sunpe**
1.  delete unreachable code
2. replace `time.Now.Sub` to `time.Since`
3. change type switch in wrapper/stmt.go

### CODE REVIEW

**Comment by zitsen** [comment]
Pelease use correct commit message style, refer to:
1. [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
2. [Confluence - PR & Commit Message Rule](https://jira.taosdata.com:18090/pages/viewpage.action?pageId=135123973)

Recommend to squash to one commit and rewrite the commit message.

File: ws/client/conn.go [hunk 1]
```diff
@@ -173,3 +174,9 @@ func (c *Client) Close() {
 		}
 	})
 }
+
+var handlerOnce sync.Once
```
Comment (huskar-t) [inline ws/client/conn.go:178]: 可能会有多个 client 不应该使用全局once

Comment (sunpe) [reply ws/client/conn.go]: 已修改

**Review by huskar-t — APPROVED** [review]
LGTM

**Pull request merged**